modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -491,25 +495,68 @@ Each configuration value is specified as a name followed by `=` and the value, a
 CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedConfig: 'region=eu-west-1' }
 ----
 
-The available configuration options are:
-
-* `file:`
-* `ftp:`
-* `http:`
-* `https:`
-* `s3:` - use this to specify the path to your S3 bucket.
-For example, `seedURI: 's3:/myBucket/myBackup.backup'`
-* `region:` - use this together with `s3:` to set the AWS region that hosts the S3 bucket.
-For example, `seedConfig: 'region=eu-west-1'`.
-
-Some seed providers may also want to pass credentials into the provider.
+`S3SeedProvider` also requires passing in credentials.
 These are specified with the `seedCredentials` option.
 Seed credentials are securely passed from the Cypher command to each server hosting the database.
 For this to work, Neo4j on each server in the cluster must be configured with identical keystores.
 This is identical to the configuration required by remote aliases, see xref:database-administration/aliases/remote-database-alias-configuration.adoc#remote-alias-config-DBMS_admin-A[Configuration of DBMS with remote database alias].
-If this configuration is not performed, the `seedCredential` option fails.
+If this configuration is not performed, the `seedCredentials` option fails.
+
+[source, cypher, role="noplay"]
+----
+CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedConfig: 'region=eu-west-1', seedCredentials: [accessKey];[secretKey] }
+----
+Where `accessKey` and `secretKey` are provided by AWS.
+
+The `CloudSeedProvider` supports:
+
+** `s3:`
+** `gs:`
+** `azb:`
+
+[.tabbed-example]
+=====
+[role=include-with-AWS-S3 label--new-5.19]
```

Comment:
I think it was introduced in 5.1.

Reply:
Yes I think `S3SeedProvider` was, but this refers to CloudSeedProvider's s3 support so should be 5.25 - will change